themes/dark/templates/editShow.mako

```diff
@@ -216,4 +220,130 @@
+                    {text: 'Skipped', value: 'Skipped'},
+                    {text: 'Ignored', value: 'Ignored'}
+                ],
+                saveStatus: '',
+                seriesLoaded: false
+            }
+        },
+        async mounted() {
+            const seriesSlug = $('#series-slug').attr('value');
+            const url = 'series/' + seriesSlug;
+            try {
+                const response = await api.get('series/' + this.seriesSlug);
+                this.seriesObj.config = response.data.config;
+            } catch (error) {
+                console.log('Could not get series info for: '+ seriesSlug);
+            }
+        },
+        methods: {
+            anonRedirect: function(e) {
+                e.preventDefault();
+                window.open(MEDUSA.info.anonRedirect + e.target.href, '_blank');
+            },
+            prettyPrintJSON: function(x) {
+                return JSON.stringify(x, undefined, 4)
+            },
+            loadSeries: async function() {
+
+            },
+            saveSeries: async function(subject) {
+                if (['seriesObj', 'all'].includes(subject)) {
+                    const data = {
+                        config: {
+                            aliases: this.seriesObj.config.aliases,
+                            dvdOrder: this.seriesObj.config.dvdOrder,
+                            flattenFolders: this.seriesObj.config.flattenFolders,
+                            anime: this.seriesObj.config.anime,
+                            scene: this.seriesObj.config.scene,
+                            sports: this.seriesObj.config.sports,
+                            paused: this.seriesObj.config.paused,
+                            location: this.seriesObj.config.location,
+                            airByDate: this.seriesObj.config.airByDate,
+                            subtitlesEnabled: this.seriesObj.config.subtitlesEnabled,
+                            release: {
+                                requiredWords: this.seriesObj.config.release.requiredWords,
+                                ignoredWords: this.seriesObj.config.release.ignoredWords
+                                // blacklist: this.seriesObj.config.release.blacklist,
+                                // whitelist: this.seriesObj.config.release.whitelist
+                            }
+                        }
+                    };
+                    try {
+                        const response = await api.patch('series/' + this.seriesSlug, data);
+                        this.saveStatus = 'Successfully patched series';
+                    } catch (error) {
+                        this.saveStatus = 'Problem trying to save series with error' + error + ', sending data: ' + data;
+                    }
+                };
+            },
+            onChangeIgnoredWords: function(items) {
+		        console.log('Event from child component emitted', items);
```

Comment:
Either remove or switch all of the `console.log` to `console.debug`.

Reply:
done